tests/func/experiments/test_show.py

```diff
@@ -471,3 +492,47 @@ def test_show_with_broken_repo(tmp_dir, scm, dvc, exp_stage, caplog):
 
     paths = ["workspace", "baseline", "error"]
     assert isinstance(get_in(result, paths), YAMLFileCorruptedError)
+
+
+def test_show_csv(tmp_dir, scm, dvc, exp_stage, capsys):
```

Comment:
I think it's okay to just test `show_experiments` with fixed `all_experiments` value, and then write a small unit test for the command to check it is passing correct parameters to `show_experiments`,

Also, for the `show_experiments`, perhaps you can just pass captured stdout to `csv.DictReader` and read from that object?

Reply:
Too much functions and commands in `dvc.command.experiments`, we should split them into separate files. Currently, it is hard to find a place for the unit test.